### PR TITLE
Create dev101.json

### PR DESCRIPTION
**Pull request by anshxlm**

### CODE REVIEW

**Review by omsenjalia — CHANGES_REQUESTED** [review]
In the "repo" section you are supposed to put your GitHub repo's link. Not the web's link

File: domains/dev101.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "An easy payment API",
+  "repo": "https://infrapay.github.io/website/",
+  "owner": {
+    "username": "InfraPay",
+    "email": "not_public@np.org",
+    "twitter": "no_twitter"
```
Comment (MaskDuck) [inline domains/dev101.json:7]: if you don't have a twitter just leave that blank.

File: domains/dev101.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "An easy payment API",
+  "repo": "https://infrapay.github.io/website/",
+  "owner": {
+    "username": "InfraPay",
+    "email": "not_public@np.org",
```
Comment (MaskDuck) [inline domains/dev101.json:6]: if you don't want to share your email, leave `""` as your email and share more of your social medias. we require at least 1 valid social media link beside github.

File: domains/dev101.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    "twitter": "no_twitter"
+  },
+  "record": {
+    "CNAME": "https://infrapay.github.io/"
```
Comment (MaskDuck) [inline domains/dev101.json:10]: cname cant have slash

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by cyberdev101** [comment]
Sorry, realized I dont need the domain anymore. Thank you for your time!

**Pull request closed without merging**